helpers/getContentImage.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+const utils = require('handlebars-utils');
+const common = require('./lib/common');
+const SafeString = require('handlebars').SafeString;
+
+const factory = globals => {
+    return function(path) {
```

Comment:
I wonder if we should create multiple helpers. Instead should we just pass `type` as an option which supports pre-defined list like `content` , `image-manager`, `carousel` etc and another flag for `src-set` as a boolean so we can return data into through a single helper. Thoughts ? cc @mattolson

Reply:
these seem like separate enough helpers (that we may even want to evolve independently) that it's nicer to have them separate to me. Just a gut feel.

I could DRY a bit better by making it call the same underlying code with two "modes" for now, though.

Reply:
@junedkazi I've extracted the common code into `lib` to reduce duplication.